include/loader/ze_loader.h

```diff
@@ -70,6 +70,18 @@ zelLoaderTranslateHandle(
 ZE_DLLEXPORT ze_result_t ZE_APICALL
 zelSetDriverTeardown();
 
+///////////////////////////////////////////////////////////////////////////////
+/// @brief Exported function for informing the loader to delay teardown of its context until the call to zelLoaderContextTeardown(). Only applies during static loader.
```

Comment:
Nit:
Only applies if application linked against static loader.
or
Only applies during static loader usage.

Reply:
Updated in latest